packages/react-search-ui-views/src/__tests__/MultiCheckboxFacet.test.js

```diff
@@ -157,6 +174,37 @@ describe("determining selected option from values", () => {
     ).toBe(false);
   });
 
+  it("will correctly determine which of the options is selected when using range filters and only the name matches", () => {
+    const wrapper = shallow(
+      <MultiCheckboxFacet
+        {...params}
+        options={rangeOptionsTimeBased}
+        values={[
```

Comment:
Just curious - could you pass `values={[rangeOptionsTimeBased[0].value]}` here instead of repeating '1000 ms ago'?